src/transformers/tokenization_utils.py

```diff
@@ -1410,3 +1410,147 @@ def clean_up_tokenization(out_string):
             .replace(" 're", "'re")
         )
         return out_string
+
+
+class FastPreTrainedTokenizer(PreTrainedTokenizer):
+    def __init__(self, **kwargs):
+        super(FastPreTrainedTokenizer, self).__init__(**kwargs)
+
+    @property
+    def tokenizer(self):
+        if self._tokenizer is None:
+            raise NotImplementedError
+        return self._tokenizer
+
+    @property
+    def decoder(self):
+        if self._decoder is None:
+            raise NotImplementedError
+        return self._decoder
+
+    @property
+    def vocab_size(self):
+        return self.tokenizer.get_vocab_size(False)
```

Comment:
Can we have a self explainable keyword argument here like `self.tokenizer.get_vocab_size(with_added_tokens=False)` or it's not possible?

Reply:
Yes, well I have a question about this. I am actually wondering, when is it interesting to get the `vocab_size` without the added tokens?

Reply:
Not in many cases I think. It was mostly because we handled the added tokens a bit differently than native tokens in python and they were added in a second time so I'll kept backward compatibility with previous code using `vocab_size`.

We can probably get rid of `vocab_size` if it's impacting the Rust code.